Photovoltaic-Cells.md

```diff
@@ -1,3 +1,25 @@
+Photovoltaic Cells generate power from sunlight.  They can connect horzontally as a free-form multiblock, even if using different tiers.  The ammount of power they generate vary throughout the day, maxing out around noon.  They must have a clear line of sight to the sky.
```

Comment:
Please add that different tiers connecting is a config value, it defaults to true for 1.10, but is false for 1.7 and 1.11.